open-machine-learning-jupyter-book/deep-learning/cnn/cnn.ipynb

```diff
@@ -2,16 +2,16 @@
  "cells": [
```

Comment:
<p>Not pulling the latest code? cnn explainer's visualization is gone</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ocademy-ai/machine-learning/pull/736/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>